arithmetization/src/main/java/net/consensys/linea/zktracer/module/oob/OobOperation.java

```diff
@@ -191,7 +191,7 @@ public OobOperation(
   private void setOpCodeFlagsAndWghtSumAndIncomingInst(MessageFrame frame) {
     final OpCode opCode = OpCode.of(frame.getCurrentOperation().getOpcode());
     // In the case of CALLs and CREATEs this value will be replaced
-    wghtSum = UnsignedByte.of(opCode.byteValue()).toBigInteger();
+    wghtSum = BigInteger.valueOf(Byte.toUnsignedInt(opCode.byteValue()));
```

Comment:
sounds a complicate way of life to get it :)

Reply:
Both of them work actually and are comparable in terms of readability. Do you have suggestions?

Reply:
at some point `wghtSum` is assigned to `oobInst`, that is expected to be a bigInteger, so even if I change the type of `wghtSum` to some more convenient, I need that conversion anyway later.

Reply:
Sorry, I am wrong. `oobInst` is now `Bytes`, so potentially we may convert both of them to `Bytes`.

Reply:
`wghtSum = Bytes.fromHexString("FF08").toUnsignedBigInteger();` due to this it cannot simply be a byte. I would suggest to leave it as it is.